crates/matrix-sdk-ui/src/timeline/event_item/content/message.rs

```diff
@@ -117,6 +117,11 @@ impl Message {
         Self { msgtype, in_reply_to, thread_root, edited, mentions }
     }
 
+    /// Create a forwarded message from a [`MessageType`].
+    pub fn forwarded(msgtype: MessageType) -> Self {
+        Self { msgtype, in_reply_to: None, thread_root: None, edited: false, mentions: None }
+    }
```

Comment:
What's a "forwarded message"? Why does it lose all the information about the message? Can you clarify all of this in the docs please?

Also, since this looks super specific to ElementX, would it make sense to do this processing there instead?

Reply:
I think I implemented this a some kind of workaround and it's no longer needed, I don't even see it referenced in my test branch for EXA. I'm deleting it.

Reply:
https://github.com/matrix-org/matrix-rust-sdk/pull/3942/commits/4b08774cb0142ea3553a7b1ab523ea1ade8eaa07

Reply:
Was this perhaps the feature where you re-send a message in another room, just like copy pasting would do.